app/src/main/java/com/android/derekbearded/Sunshine/MyViewAdapter.java

```diff
@@ -81,20 +86,18 @@ public int getItemCount() {
 
     static class MyViewHolder extends RecyclerView.ViewHolder{
 
-        ImageView icon;
-        TextView high, low, condition, date, wind, humidity;
-        ViewGroup detailView;
+        @BindView(R.id.icon) ImageView icon;
```

Comment:
This admittedly isn't much of a boilerplate reducer now that you don't have to cast the view, but I find it easier to read as you can see the layout right next to the type it returns.